services/plans/plan_states/main.go

```diff
@@ -1,10 +1,10 @@
-package plan_states
+package new_plan_states
```

Comment:
if you're just replacing the "pass plan states" by "plan states", why are you renaming it? It could just stay "plan states" no ?